packages/block-library/src/classic/editor.scss

```diff
@@ -187,6 +187,12 @@
 		color: $dark-gray-800;
 	}
 
+	// Prevent toolbar clipping on heading style in RTL languages
+	.mce-rtl .mce-flow-layout-item.mce-last {
+		margin-right: 0;
```

Comment:
Hi @draganescu, disable the margin-right rule seems to not affect the layout, is the margin-right rule necessary?

Reply:
howdy @jorgefilipecosta removing the margin-right reset will allow a previous margin-right to come into effect and hence move all the toolbar too far from the right side, causing buttons to wrap on two rows. I tested it again, to check I didn't miss use it :)

Reply:
Thank you for the clarification 👍